src/Utilities/edge.js

```diff
@@ -55,8 +56,16 @@ const inventoryHasEdgeSystems = async () => {
   return result?.data?.total > 0;
 };
 
+const getInventoryGroupDevicesUpdateInfo = async (groupUUID) => {
+  const result = await axios.get(
+    `${EDGE_API}/updates/inventory-groups/${groupUUID}/update-info`
+  );
+  return result?.data;
+};
```

Comment:
Can we place the edge api modules to /src/api/api.js or /src/api/edge directory instead? It feels like Utilities is not the right place for the API handlers

Reply:
done